awswrangler/opensearch/_utils.py

```diff
@@ -106,7 +200,92 @@ def connect(
             max_retries=10,
             retry_on_timeout=True,
         )
+        es._serverless = service == "aoss"  # type: ignore # pylint: disable=protected-access
     except Exception as e:
         _logger.error("Error connecting to Opensearch cluster. Please verify authentication details")
         raise e
     return es
+
+
+def create_collection(
+    name: str,
+    type: str = "SEARCH",  # pylint: disable=redefined-builtin
+    description: str = "",
+    encryption_policy: Optional[Dict[str, Any]] = None,
+    kms_key_arn: Optional[str] = None,
+    network_policy: Optional[Dict[str, Any]] = None,
+    vpc_endpoints: Optional[List[str]] = None,
+    boto3_session: Optional[boto3.Session] = None,
+) -> Dict[str, Any]:
+    """Create Amazon OpenSearch Serverless collection.
```

Comment:
Should we add a note to explain what actually gets created by the API call, that is default encryption and network policies and a collection.

Also, should we allow the user to pass a data access policy if they wish? If not, at least mention it in the note above?

Reply:
Yeah I agree, there's definitely space to improve docs - this was just to get started. 

I like the idea to add data policy as well but I wouldn't like to provide any "defaults" for it (unlike encryption and network policies atm).